doc/user/content/sql/functions/string_agg.md

```diff
@@ -59,14 +58,13 @@ FROM (
  a #m !z
 ```
 
-Note that in the following example, the `ORDER BY` of the subquery feeding into
```

Comment:
This comment is still correct and probably important to explain to users when `ORDER BY` matters.